com-agent-frontend/src/index.css

```
@@ -4,9 +4,610 @@
 @plugin "daisyui";
 
 :root {
-  font-family: system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+  color: #e5edf4;
+  background: #080b10;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.10.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)